polkaj-scale/src/test/groovy/io/emeraldpay/polkaj/scale/ScaleCodecReaderSpec.groovy

```diff
@@ -23,6 +24,15 @@ class ScaleCodecReaderSpec extends Specification {
         !codec.hasNext()
     }
 
+        def "Reads unsigned 64-bit long"() {
+        when:
+        def codec = new ScaleCodecReader(Hex.decodeHex("2a00000000000000"))
```

Comment:
Change this test data to be FFFFFFFFFFFFFFFF and the value to be 18446744073709551615 you will find this is -1 because you are using a long which is why we can't use them